ui/app/src/state/reducers/launcher.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2007-2021 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import GlobalState from '../../models/GlobalState';
+import { createReducer } from '@reduxjs/toolkit';
+import { INIT_LAUNCHER_CONFIG } from '../actions/launcher';
+import { deserialize, fromString } from '../../utils/xml';
+import { applyDeserializedXMLTransforms } from '../../utils/object';
+
+const reducer = createReducer<GlobalState['launcher']>(null, {
+  [INIT_LAUNCHER_CONFIG]: (state, { payload }) => {
+    let launcherConfig = null;
```

Comment:
Use consistent empty state across reducers/components.

Reply:
per conversation empty state is not needed for this one.